modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPFailureTest.java

```diff
@@ -140,6 +143,19 @@ public void addIdPWithDuplicateProperties() throws IOException {
         validateErrorResponse(response, HttpStatus.SC_BAD_REQUEST, "IDP-60025");
     }
 
+    @Test
+    public void testAddIdPWithDuplicatedPropertyKeys() throws IOException {
+
+        String body = readResource("add-idp-with-duplicated-property-keys.json");
```

Comment:
Can't we use the add-oidc-idp.json and set the duplicate property keys

Reply:
This is fixed with 2f657bedfa3cad42fd8c65f54e3b0befbc4119e0